src/test/java/no/unit/nva/model/instancetypes/InstanceTest.java

```diff
@@ -54,9 +55,10 @@ protected JsonNode generateMonographJson(String type,
                                              String pages,
                                              boolean illustrated,
                                              boolean peerReviewed,
-                                             boolean textbookContent) {
+                                             boolean textbookContent,
+                                             BookMonographContentType contentType) {
         LinkedHashMap<String, Object> instance = generateMapRepresentation(type, null, null, null, introductionBegin,
```

Comment:
Put each argument in separate line and maybe replace nulls with constants e.g. EMPTY_VOLUME, EMPTY_ISSUE, etc.